core/src/test/java/com/orientechnologies/orient/core/tx/TransactionRidAllocationTest.java

```diff
@@ -69,6 +70,43 @@ public void testAllocationCommit() {
   }
 
   @Test
+  public void testMultipleDbAllocationAndCommit() {
```

Comment:
The test is good but do not cover all cases, I mean there is no check that exception is thrown if rids do not match